SillyBusinessInc/Moldbreaker
# #276/updated animation controllers

## Purpose of this PR:
Fix the enemies getting stuck in animation hell when they collide with eachother / go in and out of attackingstate.

## How to test:
Create a legion of enemies in any branch that already supports enemies, place down as many Melee, Brute (and Ranged if available) enemies so that there is a higher likeliness of enemy collisions.
Trigger their chasing and / or attacking states.
Walk away, and keep a watchful eye on your prey.
Confirm they are no longer getting stuck in animations/ behavior.

## Links
closes #276
